pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -23,8 +23,22 @@
 , olm
 , pkg-config
 , nlohmann_json
+, voipSupport ? true
+  , gst_all_1
+  , libnice
 }:
 
+let
+  voipDeps = with gst_all_1; [
```

Comment:
Do we really need this?

Reply:
It makes the `buildInputs` a little more tidy...

Reply:
I'd guess the question is more along the lines of, in particular, “is gst-plugins-bad actually necessary?” (VoIP sounds like it should probably be in good) — to which, not having tested, I can't say anything

Reply:
It is specified as a dependency upstream and [referenced in the code](https://github.com/Nheko-Reborn/nheko/blob/9192dc8ae5b9d1a38a108633e9e33fddb4d5187e/src/WebRTCSession.cpp#L437). I think it's needed for supported video codecs.